dspace-spring-rest/src/main/java/org/dspace/app/rest/model/DSpaceObjectRest.java

```diff
@@ -20,13 +20,16 @@
  *
  */
 public abstract class DSpaceObjectRest extends BaseObjectRest<String> {
+	public static final String PRJ_DEFAULT = "";
+	public static final String PRJ_CONTEXT = "context";
```

Comment:
@tomdesair , perhaps you could add a new subtask under https://jira.duraspace.org/browse/DS-3423 to track the proper placement of these definitions.  Before figuring out where to define these values, I would like to know if they are being passed around properly.